client/src/components/game/gui/GUI.jsx

```diff
@@ -168,74 +172,13 @@ function GUI() {
             <DungeonKeys />
 
             <div className="panel-cont">
-                {shownPanel === Panels.CreateAccount && (
-                <CreateAccountPanel
-                  onCloseCallback={closePanelCallback}
-                />
-                )}
-                {shownPanel === Panels.Account && (
-                <AccountPanel
-                  onCloseCallback={closePanelCallback}
-                />
-                )}
-                {shownPanel === Panels.Settings && (
-                <SettingsPanel
-                  onCloseCallback={closePanelCallback}
-                />
-                )}
-                {shownPanel === Panels.ChangeName && (
-                <ChangeNamePanel
-                  onCloseCallback={closePanelCallback}
-                />
-                )}
-                {shownPanel === Panels.Respawn && (
-                <RespawnPanel />
-                )}
-                {shownPanel === Panels.Dungeon && (
-                <DungeonPanel
-                  onCloseCallback={closePanelCallback}
-                  dungeonPortal={targetDungeonPortal}
+                <GUIPanelWindows
```

Comment:
@Arcanorum what do you think about this?

Reply:
Seems ok. That list was getting a bit big for this component, and will get bigger.